packages/nocodb/src/modules/jobs/migration-jobs/nc_job_003_recover_links.ts

```
@@ -0,0 +1,326 @@
+import { isEE } from '~/utils';
+import { Column } from '~/models';
+
+/**
+ * This migration look for any broken link and try to recover it
+ * using the existing data
+ */
+@Injectable()
+export class RecoverLinksMigration {
+  private readonly debugLog = debug('nc:migration-jobs:recover-links');
+
+  constructor() {}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)